MetadataExtractor.sln

```diff
@@ -76,6 +76,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Specs", "Specs", "{1806EC0F
 		wiki\docs\PCX File Format.html = wiki\docs\PCX File Format.html
 		wiki\docs\PCX File Format.txt = wiki\docs\PCX File Format.txt
 		wiki\docs\QTFF.pdf = wiki\docs\QTFF.pdf
+		wiki\docs\tga_specs.pdf = wiki\docs\tga_specs.pdf
```

Comment:
Were you able to push to the wiki repo and update the submodule in this PR?

Reply:
Sorry about that. I used VS to add the solution item, but I didn't find a way to push it.

Reply:
The wiki repo is added as a git submodule to this repo, so that it can appear in VS. If you add or change files in it, they must first be pushed to the submodule repo, then this PR can update its submodule pointer (commit) to the new commit there. I think you should have push access to the wiki repo. If not, let me know where I can download this spec and I'll be sure to add it.

Reply:
I tried `git push origin HEAD:master`, but got Permission denied. I got the spec from http://tfc.duke.free.fr/coding/tga_specs.pdf

Reply:
No worries. I pushed the file to the wiki. We can update the submodule later, or feel free to do so.